<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAQwtSymbolSetWidget</class>
 <widget class="QWidget" name="SAQwtSymbolSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>137</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symbol Set Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>style:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SAQwtSymbolComboBox" name="comboBoxSymbolStyle"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFixWidthHeight">
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/icon/icons/link.png</normaloff>:/icon/icons/link.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="spinBoxSizeX">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSizeY">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Edge</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="SAPenSetWidget" name="edgetPenSet" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fill</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="SAColorPickerButton" name="colorPushButtonBrush">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderBrushAlpha">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>preview:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAQwtSymbolComboBox</class>
   <extends>QComboBox</extends>
   <header>SAQwtSymbolComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>SAColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>SAColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>SAPenSetWidget</class>
   <extends>QWidget</extends>
   <header>SAPenSetWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
